pkg/services/baremetal/host/host.go

```diff
@@ -1140,10 +1140,35 @@ func (s *Service) actionImageInstalling() actionResult {
 	// create post install script
 	postInstallScript := s.scope.HetznerBareMetalHost.Spec.Status.InstallImage.PostInstallScript
 
-	if postInstallScript != "" {
-		if err := handleSSHError(sshClient.CreatePostInstallScript(postInstallScript)); err != nil {
-			return actionError{err: fmt.Errorf("failed to create post install script %s: %w", postInstallScript, err)}
-		}
+	if postInstallScript == "" {
+		postInstallScript = "#!/bin/bash\n"
+	}
+
+	cloudInitData, err := s.scope.GetRawBootstrapData(context.TODO())
```

Comment:
we probably have to give the context now to the actionImageInstalling function right? Other option would be to store the bootstrap data somewhere, but I don't think that is valid

Reply:
@janiskemper yes, good point. I will have a look if I can get the context there somehow.

Reply:
@janiskemper I saw two ways to get the context. I asked the experts, and did it accordingly: https://www.reddit.com/r/golang/comments/1efotre/store_ctx_in_a_struct/

Reply:
@janiskemper I updated the code. OK for you?

Reply:
yes